### ISSUE TITLE
Gap between test set and validation set results

### ISSUE THREAD

**Issue by JokeroO1** [ldtho/PiFeNet#5]
Hello, I have made some minor improvements to the structure of the model. I have achieved good results on the validation set, but the results on the test set are not satisfactory. Can you tell me the reason?

**Comment by ldtho** [comment]
Achieving good results on the test set can sometimes depend on luck, as each time you train the model, it might perform differently. When submitting for the test set, one approach I personally took was to increase the number of training epochs and split the train/validation set using an 85/15 ratio.
Hope this helps!

**Comment by JokeroO1** [comment]
Thank you for your quick reply. Did the ablation experiment in the paper also use an 85:15 ratio?

**Comment by ldtho** [comment]
Sorry for the delayed reply, the ablation uses 80:20 ratio

**Comment by JokeroO1** [comment]
Thank you for your reply. @ldtho Can you tell me how many epochs were trained for the pretrained-model and how many epochs were trained for the ablation experiment?
